src/main/java/org/folio/inventory/dataimport/handlers/actions/UpdateItemEventHandler.java

```diff
@@ -161,6 +170,27 @@ public UpdateItemEventHandler(Storage storage, MappingMetadataCache mappingMetad
     return future;
   }
 
+  private Future<DataImportEventPayload> addHoldingToPayloadIfNeeded(DataImportEventPayload dataImportEventPayload, Context context, Item updatedItem) {
+    Promise<DataImportEventPayload> promise = Promise.promise();
+    if (dataImportEventPayload.getContext().get(HOLDINGS.value()) == null || StringUtils.isBlank(dataImportEventPayload.getContext().get(HOLDINGS.value()))) {
```

Comment:
StringUtils.isBlank() also checks that input param is not equals to null, so its enought just to use this method